store/extensions/G-Loader/extension.json

```
@@ -0,0 +1,38 @@
+{
+    "title": "G-Loader",
+    "description": "Simple modern user list using Qt Framework",
+    "authors": [
+      {
+        "name": "Armanta",
+        "hotel": "",
+        "username": "Armanta",
+        "discord": "Armanta#6184",
+      }
+    ],
+    "version": "1.0",
+    "categories": ["Usability"],
+
+    "source": "https://github.com/antoinechauvn/G-Loader",
+	"readme": "https://github.com/antoinechauvn/G-Loader/README.md",
+
+    "stable": true,
+
+    "framework": {
+      "name": "G-Python",
+      "version": "0.1.5"

```

You appear to have used 0.1.6 not 0.1.5
```suggestion
      "version": "0.1.6"

```